### PR TITLE
[INTERNAL] Adapt project configuration documentation

### PR DESCRIPTION
**Pull request by Thodd**
- Cachebuster docu is moved to correct location
- *.properties file encoding option is documented

### CODE REVIEW

File: docs/pages/Configuration.md [hunk 1]
```diff
@@ -87,8 +87,9 @@ Some general information:
         + `<virtual path>: <physical path>` (default `/: ./`): Any virtual path mapping can be defined here.  
           *(Only available for projects of type `module`)*  
           It is recommended that modules include their namespace in the virtual path and use the `/resources` prefix (e.g. `/resources/my/first/library/module-xy`).
-- `cachebuster`:
-    - `signatureType`: `time` or `hash`. By default, the generated cachebuster info file signatures are based on timestamps (`time`). In setups like CI environments, a mechanism based on file hashes (`hash`) might be more reliable.
+- `propertiesFileEncoding`:
```
Comment (matz3) [inline docs/pages/Configuration.md]: We should maybe also add the min. required CLI version for this (once released).

Comment (Thodd) [inline docs/pages/Configuration.md]: - moved to "resources.configuration"
- rename according to impl.

Comment (RandomByte) [reply docs/pages/Configuration.md]: Done

Comment (RandomByte) [reply docs/pages/Configuration.md]: Done

**Pull request merged**